lib/puppet/provider/rabbitmq_queue/rabbitmqadmin.rb

```diff
@@ -88,7 +89,7 @@ def create
       "--user=#{resource[:user]}",
       "--password=#{resource[:password]}",
       '-c',
-      '/etc/rabbitmq/rabbitmqadmin.conf',
+      "#{etc_dir}/rabbitmq/rabbitmqadmin.conf",
```

Comment:
given how often we use `etc_dir`, maybe we should just make `confdir` a (readonly) type parameter.